src/sentry/api_gateway/proxy.py

```diff
@@ -0,0 +1,51 @@
+"""
+Utilities related to proxying a request to a region silo
+"""
+
+
+from django.http import StreamingHttpResponse
+from requests import Response as ExternalResponse
+from requests import request as external_request
+from rest_framework.exceptions import NotFound
+from rest_framework.request import Request
+
+from sentry.models.organization import Organization
+
+# stream 0.5 MB at a time
+PROXY_CHUNK_SIZE = 512 * 1024
```

Comment:
Picked something arbitrary for now

Reply:
I've seen four kilobytes as a typical size for stream buffers in some Java libraries. Those libraries are also pretty old, so bigger seems fine to me.